KsApi/mutations/inputs/BlockUserInput.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
```

Comment:
@scottkicks Can you explain a little more about this input adapter pattern? Happy to follow it for now, but at least for a very teeny input like this, it doesn't seem obviously beneficial. I'm curious to hear more about why we've been using this pattern instead of using the `GraphAPI.XYZInput` objects directly.